infrastructure/ansible/roles/nginx/templates/etc_nginx_sites_enabled_default.j2

```diff
@@ -15,7 +15,6 @@ server {
     ssl_certificate {{ cert }} 
     ssl_certificate_key {{ cert_key }}
 
-    ssl on;
```

Comment:
This 'ssl on' directive is deprecated. This line `listen 443 ssl http2;` is enough to enable SSL